openvpn/slickvpncore/Australia-Brisbane.ovpn

```diff
@@ -0,0 +1,18 @@
+remote gw1.bne1.slickvpn.com 443 udp
+remote gw1.bne2.slickvpn.com 443 udp
+remote gw1.bne3.slickvpn.com 443 udp
+remote gw2.bne1.slickvpn.com 443 udp
+remote gw2.bne2.slickvpn.com 443 udp
+remote gw2.bne3.slickvpn.com 443 udp
+remote gw3.bne1.slickvpn.com 443 udp
+remote gw3.bne2.slickvpn.com 443 udp
+remote gw3.bne3.slickvpn.com 443 udp
+auth-user-pass /config/openvpn-credentials.txt
+client
+redirect-gateway
+remote-cert-tls server
+cipher AES-256-CBC
+proto udp
+dev tun
+nobind
+ca /etc/openvpn/slickvpn/ca.crt
```

Comment:
shouldnt this be /etc/openvpn/slickvpncore/ca.crt ??

Reply:
Perhaps.  They're the same provider, so the same CA cert.  If you prefer to keep everything segmented, a duplicate of the CA cert here and an updated reference is easy to do.

Reply:
Yes, that would be needed as the ovpn modify script will per default update the ca location and it will cause an issue if not found in the same directory.. yes, users can disable this etc but by default it should include everything in the same folder etc